kitsune/wiki/templatetags/jinja_helpers.py

```diff
@@ -16,3 +17,10 @@ def diff_table(content_from, content_to):
 @library.global_function
 def generate_video(v):
     return Markup(parser.generate_video(v))
+
+
+@library.global_function
+def is_switching_devices_document(doc):
+    return (
+        doc.parent.slug if doc.parent else doc.slug
```

Comment:
shall we make this a property on the model?

Reply:
Yes, that's better, thanks @akatsoulas.